components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/impl/BasicAuthAuthenticator.java

```diff
@@ -63,13 +63,13 @@ public boolean authenticate(Request request, Response responder, ServiceMethodIn
                     return true;
                 }
             } else {
-                throw new APIMgtSecurityException("Missing Authorization header in the request.`",
-                        ExceptionCodes.INVALID_AUTHORIZATION_HEADER);
+                throw new APIMgtSecurityException("Missing 'Authorization : Basic' parameter in the request.`",
```

Comment:
I think that's used in general for any value. But in our case, since we know it's a header, we can tell that.